README.md

```diff
@@ -4,60 +4,32 @@
 [![CircleCI](https://circleci.com/gh/ethereum/remix/tree/master.svg?style=svg)](https://circleci.com/gh/ethereum/remix/tree/master)
 [![Documentation Status](https://readthedocs.org/projects/docs/badge/?version=latest)](https://remix-ide.readthedocs.io/en/latest/index.html)
 
-Ethereum tools for the web.
 
-*Are you looking for the Remix IDE? Follow [this link](https://github.com/ethereum/remix-ide)!*
+**Remix** is a suite of tools which helps in smart contract development, compilation, testing & deployment. These tools also works as a core of native plugins of Remix IDE.
```

Comment:
**Remix** is a suite of tools ~which~ _that_ helps ~in~ smart contract development, compilation, testing & deployment. These tools also work~s~ as a core of native plugins of Remix IDE.